### DISCUSSION TITLE
Why does ILSpy use my Microphone?

### DISCUSSION THREAD

**Discussion by ThaDaVos** [icsharpcode/ILSpy#3228] [General] [answered]
As the title suggests, why does ILSpy use my Microphone?
![afbeelding](https://github.com/icsharpcode/ILSpy/assets/5251028/92e71ce4-cb2b-47cd-8b0d-e4cb83b792fb)
![afbeelding](https://github.com/icsharpcode/ILSpy/assets/5251028/0f2ac23e-7df4-42d0-8bcd-d0e54bf6af3c)

**Comment by christophwille** [comment]
Why does that have the VSCode logo?

**Reply by ThaDaVos** [reply]
That's my curiousity too - I am checking my VScode to see if I can find something

**Comment by ThaDaVos** [comment, marked as answer]
![afbeelding](https://github.com/icsharpcode/ILSpy/assets/5251028/738c7506-e117-4a0e-af30-db3bbaa622cc)
Seems this is the cause - why it reports it as ILSpy is still weird though
